server/index.ts

```diff
@@ -1,29 +1,30 @@
-const express = require('express');
+import { Pet, DetailedPet } from 'swagger-data/model/models';
```

Comment:
I believe all of the models are exported from the root file in `swagger-data`. So `import { Pet, DetailedPet } from 'swagger-data';` may work also